config

```diff
@@ -198,6 +198,22 @@
 #has_dep=[BUNDLE]
 
 
+[bundle-info]
+
+#
+# Options for the "swupd bundle-info" command
```

Comment:
Are we missing the 3rd-party repo add, delete and list here?

Reply:
well, this PR sits on top of other PRs (that still have not merged), so you are looking at more that what this PR brings to the table. I forgot to add the disclaimer in the PR details. I'll add it to avoid confusions.

Reply:
I just added this disclaimer "This PR is built on top of #1232, #1233 and #1234 and have to be reviewed first".

Reply:
yes, I didn't included "add, remove and list" to the config file because they have no flags.